### PR TITLE
Introduce a team of code owners for the CI system.

### PR DESCRIPTION
**Pull request by Zimmi48**
This means that all the members of the team will receive a review request for PRs on the CI, but only one of them will need to approve the PR, and this will remove the review request for the others.

Currently the team contains Emilio and Gaetan, the two current code owners of these files. It makes sense to start experimenting on this component since they had already decided to make their role symmetric.

Updating the list of maintainers can be done by updating the list members, and without changing the CODEOWNER file.

If this is seen as a good idea, we can slowly convert the rest of the CODEOWNER file using this style (add a documentation-maintainers team, build-system-maintainers team, etc.).

### CODE REVIEW

**Comment by SkySkimmer** [comment]
>This means that all the members of the team will receive a review request for PRs on the CI, but only one of them will need to approve the PR, and this will remove the review request for the others.

Do you know if it works when the PR is from a member of the team?

**Review by ejgallego — COMMENTED** [review]
works for me

**Comment by Zimmi48** [comment]
> Do you know if it works when the PR is from a member of the team?

Good question. Do you want to get the answer before approving this? (I can try doing some tests in another repository.) Otherwise I guess we'll see, it shouldn't be much different than the situation today when the principal maintainer submits a PR...

**Review by SkySkimmer — APPROVED** [review]

**Review by ejgallego — APPROVED** [review]

**Comment by maximedenes** [comment]
> Good question. Do you want to get the answer before approving this? (I can try doing some tests in another repository.) Otherwise I guess we'll see, it shouldn't be much different than the situation today when the principal maintainer submits a PR...

I think it would be good to test that.

**Comment by Zimmi48** [comment]
> I think it would be good to test that.

This is requiring me more work, so unless you see a real problem, I'd prefer to merge as-is.

**Review by maximedenes — APPROVED** [review]
Ok, we'll see what it gives in practice.

**Comment by ejgallego** [comment]
So now it turns out that I am getting a review request on my own PRs; I don't see a way to remove that other than removing the team.

This does indeed a bit of noise unfortunately to my review requested list :(

**Comment by Zimmi48** [comment]
So it looks then like the answer to this question:

> Do you know if it works when the PR is from a member of the team?

was yes, it even works too often.

Anyways, IMO this is a GitHub-bug, so I'm going to report it. Hopefully they fix it and we can expand the use of this feature.

**Comment by maximedenes** [comment]
> So now it turns out that I am getting a review request on my own PRs

I would be curious to see how you review your PRs, and how you reply to your reviews :)

**Comment by Zimmi48** [comment]
You can leave review comments on your own PRs. But if you do this, it will remove the request for review from the rest of the team.

**Comment by Zimmi48** [comment]
@ejgallego Could you test and confirm what I've just said? If it's really the case, I'll follow up on my GitHub bug report because this is another issue.

**Comment by ejgallego** [comment]
> @ejgallego Could you test and confirm what I've just said? If it's really the case, I'll follow up on my GitHub bug report because this is another issue.

I can comment, but not approve or denegate; after commenting the review request is still shown.

**Comment by SkySkimmer** [comment]
If I'm the one commenting it goes away as usual.

**Comment by Zimmi48** [comment]
> I can comment, but not approve or denegate; after commenting the review request is still shown.

OK, so at least that is good.

**Pull request merged**
